### PR TITLE
Move tests to catch2

### PR DESCRIPTION
**Pull request by sveinung-r**

### CODE REVIEW

**Comment by ghost** [comment]
Can one of the admins verify this patch?

**Comment by ReedOnly** [comment]
test this please

File: lib/tests/load.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#include <catch/catch.hpp>
+
+#include <ert/util/test_util.hpp>
+#include <nex/private/util.hpp>
+
+TEST_CASE( "load" "[load]") {
```
Comment (JensGM) [inline lib/tests/load.cpp]: Missing comma, use `TEST_CASE( "load" "[load]")`

File: lib/tests/spe1.cpp [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+            "QOP", "QGP", "QWP", "QOI", "QGI", "QWI" , "COP", "CGP",
+            "CWP", "COI", "CGI", "CWI", "API", "WCUT", "GOR", "WOR",
+            "Q1P", "Q1I", "Q2P", "Q2I"
+        }}
+    }};
+
+    const auto spe1 = load("test-data/SPE1.plt");
+}
+
+SCENARIO( "spe1" "[spe1]") { WHEN( "loading nexus plot file" ) {
```
Comment (JensGM) [inline lib/tests/spe1.cpp]: Missing comma

File: lib/tests/units.cpp [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                             nex::UnitSystem::Measure measure,
+                             float n,
+                             float expected_conversion ) {
+    float c = n * u.conversion( measure );
+    CHECK( n == Approx(c / expected_conversion) );
+}
+
+}
+
+TEST_CASE( "unit conversions" "[units]") {
```
Comment (JensGM) [inline lib/tests/units.cpp]: Missing comma

**Review by JensGM — CHANGES_REQUESTED** [review]
I found 3 missing commas, you should take a quick look incase there are more.

**Pull request closed without merging**